docker-credential-mmds/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/firecracker-microvm/firecracker-containerd/docker-credential-mmds
 
-go 1.16
+go 1.17
```

Comment:
you will probably need to run `go mod tidy` in all the places where you are updating go.mod.

There is a script that automates this in [tidy.sh](https://github.com/firecracker-microvm/firecracker-containerd/blob/main/tools/tidy.sh). Running `make tidy` also executes this script https://github.com/firecracker-microvm/firecracker-containerd/blob/main/Makefile#L135-L136

Reply:
thanks for the clarification @swagatbora90, I just updated the PR